src/txdb.cpp

```diff
@@ -758,25 +763,25 @@ CacheConfig CacheSizeCalculations(int64_t _nTotalCache)
     _nTotalCache -= cache.nBlockTreeDBCache;
     if (BLOCK_DB_MODE == LEVELDB_BLOCK_STORAGE)
     {
-        // use up to 5% for the level db block cache but no bigger than 256MB
-        cache.nBlockDBCache = _nTotalCache * 0.05;
+        // use up to 10% for the level db block cache but no bigger than 256MB
+        cache.nBlockDBCache = _nTotalCache * 0.10;
         if (cache.nBlockDBCache < cache.nBlockTreeDBCache)
             cache.nBlockDBCache = cache.nBlockTreeDBCache;
-        else if (cache.nBlockDBCache > 256 << 20)
-            cache.nBlockDBCache = 256 << 20;
+        else if (cache.nBlockDBCache > 1024 << 20)
+            cache.nBlockDBCache = 1024 << 20;
 
-        // use up to 1% for the level db undo cache but no bigger than 64MB
-        cache.nBlockUndoDBCache = _nTotalCache * 0.01;
+        // use up to 2% for the level db undo cache but no bigger than 64MB
```

Comment:
Also here it seems we need to update the comment with 128

Reply:
done